js/9exercise.js

```diff
@@ -0,0 +1,8 @@
+const  word = 'space';
+
+const wordSplit = word.split('')
+const indexA = wordSplit.indexOf('a')
+const splice = wordSplit.splice(indexA, 1, 'i')
+
+console.log('wordSplit', wordSplit)
```

Comment:
Definitely works. Recommend resolving this with built in string function .replace()